src/common/check_dataset_schema/test.py

```diff
@@ -0,0 +1,100 @@
+import subprocess
+from os import path
+import json
+
+input_path = meta["resources_dir"] + "resources_test/common/pancreas/dataset.h5ad"
```

Comment:
If the other suggestion gets accepted.

```suggestion
input_path = meta["resources_dir"] + "/pancreas/dataset.h5ad"
```